src/Accordion.php

```diff
@@ -154,6 +155,19 @@ public function renderItems()
         return implode("\n", $items);
     }
 
+    /**
+     * check wether accordion is expanded
+     * This method will check if expand properties exist and determine value of this properties
+     * 
+     * @return bool
+     */
+    private function _isAccordionExpanded($item){
+        if(isset($item['expand'])){
```

Comment:
`if (isset($item['expand'])) {`

Reply:
Fix in this https://github.com/yiisoft/yii2-bootstrap4/pull/196/commits/65d25c75c39e69766b38f2fdd1e292b72325c546